.gitignore

```diff
@@ -87,4 +87,8 @@ ignite/
 *.classpath
 *.project
 *.prefs
-*.factorypath
\ No newline at end of file
+*.factorypath
+
+# python cache
```

Comment:
Scripts looks pretty comprehensive. We do need a comprehensive README as well.

Some remaining comments to be resolved:
(1) https://github.com/futurewei-cloud/alcor/pull/569#discussion_r600063256
(2) https://github.com/futurewei-cloud/alcor/pull/569#discussion_r600064466
(3) https://github.com/futurewei-cloud/alcor/pull/569#discussion_r600073125
(4) https://github.com/futurewei-cloud/alcor/pull/569#discussion_r600090468

Reply:
1. A readme file is on its way.
2. The names are taken as per the documentation.
3. At this stage it is not possible. Moreover we want to keep the tests act like end user and be independent.
4. Goal state parameters will be going into the config file.

Reply:
Readme updated.